lib/cosmos/tools/script_runner/script_runner_frame.rb

```diff
@@ -1042,9 +1042,9 @@ def toggle_disconnect(config_file)
         dialog_layout = Qt::VBoxLayout.new
 
         chooser = FileChooser.new(self, "Config File", config_file, 'Select',
-                                  File.join('config', 'tools', 'cmd_tlm_server', config_file))
+                                  File.dirname(config_file))
         chooser.callback = lambda do |filename|
-          chooser.filename = File.basename(filename)
+          chooser.filename = filename
```

Comment:
Also fixed this bug in Script Runner while I found the same one in Command Sequence